src/components/media/MediaMenu.tsx

```diff
@@ -33,13 +35,15 @@ const MediaMenu = ({
     isDisabled = false,
     dropdownProps = {},
     menuProps = {},
+    intl,
     ...rest
 }: MediaMenuProps) => (
     <DropdownMenu constrainToScrollParent isRightAligned {...dropdownProps}>
         <PlainButton
             isDisabled={isDisabled}
             className={classnames('bdl-Media-menu', className)}
             type={ButtonType.BUTTON}
+            aria-label={intl.formatMessage(messages.commentOptionsArialLabel)}
```

Comment:
Very minor, but we usually try to alphabetize props, object properties, etc. when possible.

Reply:
but don't know how it will look after alphabetize this property

Reply:
you mean to move aria-label to be the first prop?